receiver/mongodbatlasreceiver/receiver.go

```diff
@@ -53,9 +53,9 @@ func newMongoDBAtlasScraper(log *zap.Logger, cfg *Config) (scraperhelper.Scraper
 func (s *receiver) scrape(ctx context.Context) (pdata.Metrics, error) {
 	var start time.Time
 	if s.lastRun.IsZero() {
-		start = s.lastRun
-	} else {
 		start = time.Now().Add(s.cfg.CollectionInterval * -1)
+	} else {
```

Comment:
This change is unrelated to the one described in the PR. Should there be a separate line item in the changelog for it?

Reply:
Whoops, got my branches mixed up, thanks 🙂